src/library_gl.js

```diff
@@ -715,9 +715,22 @@ var LibraryGL = {
     switch(name_) {
       case 0x1F00 /* GL_VENDOR */:
       case 0x1F01 /* GL_RENDERER */:
-      case 0x1F02 /* GL_VERSION */:
         ret = allocate(intArrayFromString(GLctx.getParameter(name_)), 'i8', ALLOC_NORMAL);
         break;
+      case 0x1F02 /* GL_VERSION */:
+        var glVersion = GLctx.getParameter(GLctx.VERSION);
+        // return GLES version string corresponding to the version of the WebGL context
+#if USE_WEBGL2
+        if (GLctx.canvas.GLctxObject.version >= 2) {
```

Comment:
I have another argument for using `GLctx.canvas.GLctxObject.version` to make the GL_VERSION string, this value correspond to the GL ES version API which is effectively implemented by Emscripten, it is this number that is used in the code to select which GL ES feature is available, not the WebGL version string.

I propose to remove the GLSL part of the patch and only use the context version to make the GL_VERSION string.